setup.py

```diff
@@ -25,7 +25,8 @@
     ],
     packages=[
         'edxlearndot',
-        'tests'
+        'edxlearndot.migrations',
```

Comment:
@pomegranited oh yeah I didn't notice this, but I think it would be better to use find_packages instead of add each individual package, could you please improve?

Reply:
Sure thing -- see fc84e8d26ee52115d072d8dbb0e45b37617212f6